### ISSUE TITLE
Needs Spark rename updates

### ISSUE THREAD

**Issue by annegentle** [CiscoDevNet/devnet-express-code-samples#26]
Spark renamed to Webex Teams, and use `ACCESS_TOKEN` instead of `SPARK_TOKEN` - plus look for sparkbot.io renamed to webex.bot, and developer.ciscospark.com redirecting to developer.webex.com.

**Comment by annegentle** [comment]
Hey Ashley (@aroach ) I did the content changes in the devnet-express repo, but haven't circled back here. Can you take it?

**Comment by aroach** [comment]
@annegentle are we only changing `ACCESS_TOKEN` -- looks like there are a lot of function names and variables with spark in them as well...

**Comment by annegentle** [comment]
Yeah I’m working towards removing Spark from variable names as well, using Webex or Webex Teams with matching capitalization or kebab-case as needed... see what you think of that as you dive in.

**Comment by annegentle** [comment]
See what you think of this PR for the DCI code: https://github.com/CiscoDevNet/devnet-express-dci-code-samples/pull/6

**Comment by aroach** [comment]
ok, what about filenames too?  such as `spark-room-easy.py`

**Comment by annegentle** [comment]
That's what I'm working through today... trying to move those to `webex-teams-room-easy.py` for example. Will require testing, yeah.

**Comment by aroach** [comment]
I think i got all of the changes except the meraki mission.  See PR #27
